### PR TITLE
feat: graph:library production with persisted queries

### PR DESCRIPTION
**Pull request by sgrove**
🎉 Thanks for submitting a pull request! 🎉

#### Summary

Fixes:
 - netlify/labs#36
 - netlify/labs#35
 - Generated library code didn't specify a `schemaId`

Features:
 - Adds a production mode for `graph:library` that persists queries with various levels of caching and `schemaId` pinning
 - Adds a `graph:init` that creates an env var for webhook secret signing and a token for creating persisted queries and modifying schemas
 - Normalizes GraphQL operations document on `graph:library` to help minimize merge conflicts so multi-user teams have a better experience

---

For us to review and ship your PR efficiently, please perform the following steps:

- [ ] Update or add tests (if any source code was changed or added) 🧪
- [ ] Update or add documentation (if features were changed or added) 📝
- [ ] Make sure the status checks below are successful ✅

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,6 +91,7 @@
     "Robin Hellemans <robin@oreon.io>",
     "Rodney Coyer <rodney.coyer@gmail.com>",
     "Rohit J (https://twitter.com/syntaxhacker0)",
+    "Roneo.org (Roneo.org)",
```
Comment (erezrokah) [inline package.json:94]: Side effect of https://github.com/netlify/cli/pull/4442, you can ignore

**Comment by erezrokah** [comment]
@sgrove what is the status of this PR?

**Comment by sgrove** [comment]
@erezrokah It's still a WIP with some changes that have to happen on Netlify's frontend and a new build plugin.

**Comment by danez** [comment]
@sgrove Is this still valid? Or can we close this PR?

**Comment by sgrove** [comment]
Go ahead and close it

**Pull request closed without merging**